svf/lib/Util/Options.cpp

```diff
@@ -679,6 +679,14 @@ const Option<bool> Options::PrintPathCond(
 );
 
 
+// SaberSVFGBuilder.cpp
+const Option<bool> Options::IgnoreExtFun(
+    "saber-ignore-ext-fun",
+    "Don't include pointers returned by external function during collecting globals",
+    false
```

Comment:
make it true as default.

Reply:
Making it true as default will cost performance downgrade, should we do some profiling first?

Reply:
See my new name. The name should be used the other way around to make it clear. `True` is your current false.

Reply:
Ok